test/api/python/provisioner/test_sw_upgrade.py

```diff
@@ -90,7 +90,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 @pytest.mark.unit
-def test_get_swupgrade_info():
+def test_get_swupgrade_info(tmpdir_function):
```

Comment:
Who is responsible to provide this `tmpdir_function` function?

Reply:
it's one of our custom fixtures

Reply:
So if I understand correctly, is it safe to miss/do not specify any parameters(fixtures) in the test function definition?

Reply:
yes, sure. If no fixtures are mentioned pytest will not do any preparation steps (but there might be [auto applied](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)) fixtures